packages/storybook-react/src/stories/Button/ButtonAppearances.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { Button } from '@utrecht/component-library-react';
+import React from 'react';
+import { argTypes } from './argTypes';
+
+export default {
+  title: 'React.js Component/Button/Appearances',
+  id: 'react-button-appearances',
+  component: Button,
+  argTypes,
+} as ComponentMeta<typeof Button>;
+
+const Template: ComponentStory<typeof Button> = (args) => <Button {...args} />;
+
+export const Primary = Template.bind({});
```

Comment:
Besides `primary-action-button`, `secondary-action-button` and `subtle-button` there is also the default without `appearance` that should be included.